src/sentry/api/endpoints/release_thresholds/health_checks/is_crash_free_rate_healthy.py

```diff
@@ -124,7 +124,7 @@ def is_crash_free_rate_healthy_check(
     except IndexError:
         return False, -1
 
-    crash_free_percent = (1 - (crash_count / total_count)) * 100
+    crash_free_percent = (1 - (crash_count / max(total_count, 1))) * 100
```

Comment:
Out of curiosity, when would we expect `total_count` to be less than 1?

Reply:
It would never be less than 1
but if the fetch has an issue - it could return 0 which would raise a `divide by 0` issue

Reply:
ah ok makes sense